mindmap/static/js/src/jsmind.draggable-node.js

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
This is a no for me. Maintaining a file this way is more complex than using a CDN. Imagine the work of updating this instead of just changing the version of the CDN. Also, how often is the CDN repository down to consider better downloading it? Can we reconsider this decision? Thanks
cc @johnvente

Reply:
CDN's can go down anytime, it's not something usual but can happen. 

@BryanttV use cdn's again please make sure that instead of having the version put this `@latest` which means if you have 
`https://cdn.jsdelivr.net/npm/jsmind@0.6.4/es6/jsmind.min.js` replace it for this `https://cdn.jsdelivr.net/npm/jsmind@latest/es6/jsmind.min.js` 

`@latest`  will install always the last version

Reply:
That'd be better! Thanks.